### ISSUE TITLE
Update to FOP 2.8

### ISSUE THREAD

**Issue by rdorris** [cajuncoding/ApacheFOP.Serverless#9]
Have you done any research or have any guidance on updating to version FOP 2.8? I'm not proficient in the java environment, but do you think it's just a matter of recompiling changing the libraries from 2.6 to 2.8?
Thanks!

**Comment by cajuncoding** [comment]
Yes it’s quite easy! I’ve updated the project several times, but didn’t realize a new version was released.

Just like .Net Nuget, the most used package & version management in Java is called _Maven_.

All I have to do is update the Maven reference and re-run the lifecycle to pull the latest and re-compile; and do some smoke testing.

But it’s highly unlikely anything in the interface or configuration structure has changed.

I will try to do this tomorrow…

**Comment by rdorris** [comment]
Here are the upgrades I did, and it seems to be working fine:
azure-functions-java-library 3.0.0
fop 2.8
commons-lang3 3.12.0
commons-text 1.10.0
commons-io 2.13.0

Let me know if there are any upgrades you think should be included.
